tools/depends/native/python36-native/Makefile

```diff
@@ -11,6 +11,10 @@ CWD=$(shell pwd)
 HOSTPYTHONDIR=$(CWD)/$(PLATFORM)/hostpython
 
 CONFIGURE=./configure --prefix=$(NATIVEPREFIX) --disable-shared --disable-framework --without-pymalloc
+ifeq ($(ENABLE_WAYLAND),yes)
+  CONFIGURE += --with-system-expat
```

Comment:
does passing this argument unconditionally causes an error?
If you want it explicit just ignore my comment.

Reply:
I prefer to make stuff explicit